language-design.md

```diff
@@ -70,6 +205,12 @@ what ways do the themes apply to the study and creation of DSLs?
 
 **Response**
 
+The language stereotypes explored in the Yang and Rabkin article are very much applicable to the use of DSL’s. People who use DSL’s are susceptible to this same “language-based snobbery”. Consider the example of jQuery. In one of Julien’s previous internships, jQuery was scoffed at – although there were performance-related improvements to its eventual removal. Even though it is widely used and no doubt simplifies interactions with HTML, some programmers may think of it as a crutch, and a sign of weakness as it removes the need to understand the DOM on a deeper level. 
```

Comment:
This is a super interesting story, and I would have enjoyed to hear more about the experience. Were any of their critiques correct or necessary? You mentioned that there were eventually performance-related reasons why they moved away from jQuery, but did the co-workers ever mention that or was it just called a "crutch?" In the same vain, did they justify the necessity of understanding the DOM on a deeper level? Although I think your team found an experience of snobbery, it's also important to understand the very thin line between snobbery and critiques. When should we listen to people who criticize our language choice and when should we stick to our guns?